### PR TITLE
New crispy forms layout helpers

### PR DESCRIPTION
**Pull request by biyeun**
again, to alleviate the conflicts on staging

### CODE REVIEW

File: corehq/apps/style/crispy.py [hunk 1]
```diff
@@ -67,3 +68,78 @@ def render(self, form, form_style, context, template_pack=None):
...
+        if not hasattr(self, 'attrs'):
+            self.attrs = {}
+
+        if 'css_class' in kwargs:
+            if 'class' in self.attrs:
+                self.attrs['class'] += " %s" % kwargs.pop('css_class')
+            else:
+                self.attrs['class'] = kwargs.pop('css_class')
+
+    def render(self ,form, form_style, context, template_pack=None):
```
Comment (dimagimon) [inline corehq/apps/style/crispy.py]: E203 whitespace before ','

Comment (dimagimon) [inline corehq/apps/style/crispy.py]: E231 missing whitespace after ','

**Comment by biyeun** [comment]
@NoahCarnahan Good question, and i'll be sure to clarify it in the docs / styleguide.

It's basically a shortcut for `crispy.HTML(render_to_string("some_template.html", { 'foo': 'bar'})`, and yep that's intentional

**Comment by NoahCarnahan** [comment]
Cool, looks gtm. Any need to wait for tests?

**Comment by biyeun** [comment]
@NoahCarnahan nope, we don't have any ui tests for this so...

**Pull request merged**